cli_reference/get_started_cli.adoc

```diff
@@ -113,6 +113,17 @@ directory on your PATH. To check your PATH, open the Command Prompt and run:
 C:\> path
 ----
 
+ifdef::openshift-origin,openshift-enterprise[]
+[NOTE]
+====
+The `oc` command provides command line tools for managing applications, while
+the `oadm` command provides tools for managing a cluster. The `openshift` binary
+is an all-in-one command that can be used for starting services that make up the
+{product-title} cluster. The `openshift` binary can perform all the same actions
+as the `oc` and `oadm` commands.
+====
+endif::[]
+
```

Comment:
@xingxingxia I initially added this here because I thought it was worth mentioning after discussion of the `oc` binary just before. However, I can remove it from this page.